packages/blog2/src/components/mdx/MdxPre/index.module.css

```diff
@@ -25,6 +25,9 @@
 }
 
 .pre {
+  font-family: Menlo, Monaco, "Courier New", monospace;
+  font-size: 12px;
```

Comment:
there are styles for VSCode by default, which I want to see in the `<pre>` elements